### PR TITLE
Rnnt

### PR DESCRIPTION
**Pull request by G4V**

### CODE REVIEW

File: package/dataset-librispeech/install.sh [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  if [ "${MD5_CALC:0:32}" != "${MD5_EXP}" ] ; then
+    echo "Error: MD5 of ${ARCH} is incorrect!"
+    exit 1
+  fi
+}
+
+function uncompress {
+  ARCH=${1}
+  echo ""
+  echo "Utarring archive ..."
```
Comment (psyhtest) [inline package/dataset-librispeech/install.sh:31]: Untarring

File: package/dataset-librispeech/install.bat [hunk 2]
```diff
@@ -0,0 +1,12 @@
+
+echo Downloading COCO 2017 validation dataset
```
Comment (psyhtest) [inline package/dataset-librispeech/install.bat:2]: Remove `install.bat` altogether as not supported?

File: package/dataset-librispeech/install.sh (hunk 1 above)
Comment (G4V) [reply package/dataset-librispeech/install.sh]: Done

File: package/dataset-librispeech/install.bat (hunk 2 above)
Comment (G4V) [reply package/dataset-librispeech/install.bat]: Done

**Pull request merged**